ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -53,23 +53,23 @@
 {# mcs client #}
 {% include 'eos/mcs-client.j2' %}
 {# match-list #}
-{% include 'eos/match-lists.j2' %}
+{% include 'eos/match-list-input.j2' %}
 {# as-paths #}
 {% include 'eos/as-path.j2' %}
 {# macsec #}
 {% include 'eos/mac-security.j2' %}
 {# hostname #}
 {% include 'eos/hostname.j2' %}
 {# domain lookup #}
-{% include 'eos/domain-lookup.j2' %}
+{% include 'eos/ip-domain-lookup.j2' %}
 {# name servers #}
-{% include 'eos/name-servers.j2' %}
+{% include 'eos/name-server.j2' %}
 {# dns domain #}
 {% include 'eos/dns-domain.j2' %}
 {# domain-list #}
 {% include 'eos/domain-list.j2' %}
 {# Object Tracking #}
```

Comment:
any wish to also sanitize the comments or we can ignore for now?

Reply:
Good catch, I will clean up the comments!

Reply:
updated in 2f4128a